### ISSUE TITLE
Modifying Inventory Name Bug

### ISSUE THREAD

**Issue by DelayedGaming** [SkriptLang/Skript#5215]
### Skript/Server Version

```markdown
[Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[Skript] Server Version: git-Paper-81 (MC: 1.19)
[Skript] Skript Version: 2.6.3
[Skript] Installed Skript Addons:
[Skript]  - skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect)
[Skript] Installed dependencies:
[Skript]  - Vault v1.7.3-b131
```

### Bug Description

Abnormal behaviour was observed whilst cloning an inventory and modifying both inventory name.
Refer below for a walkthrough:

### Expected Behavior

Indefinable.. Refer below for a walkthrough:

### Steps to Reproduce

[This snippet](https://hastebin.com/unevupaban.sql) is used to show the bug in action.

1. Run command `/c1` -- A chest inventory named "C1" with slot 0 as GRAY DYE will appear.
2. Run command `/clonechest` -- This clones Chest C1 `{-c1}` to Chest C2 `{-c2}`, renamed Chest C1 to "C1 Renamed", renamed Chest C2 to "C2" and appended a LIME DYE to slot 1 in C2. A debug message is also printed to validate Chest C2 is exactly the same as Chest C1
3. Run command `/c2` -- Chest C2 will appear:
- **Expected:** A chest inventory named "C2" with slot 0 and 1 as GRAY DYE and LIME DYE respectively.
- **Result:** A chest inventory named _"C1"_ with slot 0 and 1 as GRAY DYE and LIME DYE respectively.
4. Run command `/c1` to reveal Chest "C1" and visually verify its change:
- **Expected:** A chest inventory named "C1 Renamed" with slot 0 and 1 as GRAY DYE and LIME DYE respectively.
- **Result:** A chest inventory named _"C1"_ with _slot 0 and 1 as GRAY DYE and AIR respectively_.

FYI: Reproducing this but with a **player inventory** returns expected results:
![image](https://user-images.githubusercontent.com/72163224/202516582-6bf8de50-97ab-4083-b3c8-4ed468c8603e.png)

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by DelayedGaming** [comment]
TLDR version:
- Set var1 to chest inventory
- Set var2 to var1
- Change inventory name of either var1 or var2
With the above procedure, var1 and var2 will desynced, having 2 different inventories, as Skript clones the inventory with a new name -- Directly changing it is not possible

**Comment by TheLimeGlass** [comment]
Please provide the Skript coding for the inventories.

**Comment by DelayedGaming** [comment]
1. Set a variable to a new chest inventory
`set {-c1} to chest inventory named "Chest 1"`
![pscreatec1](https://user-images.githubusercontent.com/72163224/210869382-a434a894-650f-4626-b3c7-08601adeda1a.png)
2. Set a second variable to previous declared variable
`set {-c2} to {-c1}`
3. Change name of inventory saved in second variable to any name of your choice
`set name of {-c2} to "Chest 2 :)"
4. Open inventory saved in second variable
`open {-c2} to player` -- Expected: Chest inventory named "Chest 2 :)"
![pssetc2toc1](https://user-images.githubusercontent.com/72163224/210869445-54c1fca0-1390-4e08-98fb-3fede8be59bf.png)

**Comment by TheLimeGlass** [comment]
Issue is when setting the name of an inventory, the ExprName only applies when the inventory is open. The changer does not take into consideration a variable inventory.
